FlightReservation/components/formComponent.js

```diff
@@ -0,0 +1,180 @@
+                    }></TextInput>
+                </View>
+            }
+            <View style={{ flexDirection: 'row' }}>
+                <Text style={styles.inputHeader}>Email</Text>
+                <Text style={validEmail ? { color: "gray" } : styles.errorText}>*</Text>
+                {!validEmail && <Text style={styles.errorText}>Email in use. Use a different email</Text>}
+            </View>
+            <TextInput
+                style={[styles.inputStyle, { borderColor: borderColor.borderColorEmail }]}
+                value={objValues.email}
+                onChange={(e) =>
+                    checkInputsIsNull(e, 'email', 'borderColorEmail')
+                }></TextInput>
+            <View style={{ flexDirection: 'row' }}>
+                <Text style={styles.inputHeader}>Password</Text>
+                <Text style={validPassword ? { color: "gray" } : styles.errorText}>*</Text>
+                {!validPassword && <Text style={styles.errorText}>Incorrect email and/or password</Text>}
+            </View>
+            <View style={styles.containerPassword}>
+                <TextInput
+                    style={[
+                        styles.inputPassword,
+                        {
+                            borderTopColor: borderColor.borderColorPassword,
+                            borderBottomColor: borderColor.borderColorPassword,
+                            borderLeftColor: borderColor.borderColorPassword,
+                        },
+                    ]}
+                    secureTextEntry={isSeePassword}
+                    value={objValues.password}
+                    onChange={(e) =>
+                        checkInputsIsNull(e, 'password', 'borderColorPassword')
+                    }
+                />
+
+                <View
+                    style={[
+                        styles.containerIconPassword,
+                        {
+                            borderRightColor: borderColor.borderColorPassword,
+                            borderTopColor: borderColor.borderColorPassword,
+                            borderBottomColor: borderColor.borderColorPassword,
+                        },
+                    ]}>
+                    <Pressable onPress={changeSeePassword}>
+                        <Image
+                            style={[
+                                styles.iconSeePassword,
+                                { tintColor: borderColor.borderColorPassword },
+                            ]}
+                            source={require('../img/ojo-grey.png')}
+                        />
+                    </Pressable>
+                </View>
+            </View>
+            <Text style={styles.instruccionsPassword}>
+                Use 8 or more characters with a mix of letters, numbers, and symbols
+        </Text>
+        </View>
+    );
+}
+const styles = StyleSheet.create({
+    inputHeader: {
+        fontSize: 16,
+        color: '#818181',
```

Comment:
all the colors have to be inside constants file

Reply:
done!